src/txmempool.h

```diff
@@ -565,7 +565,12 @@ class CTxMemPool
     void ApplyDelta(const uint256 hash, CAmount &nFeeDelta) const;
     void ClearPrioritisation(const uint256 hash);
 
-public:
+    /** Get the transaction in the pool that spends the same prevout */
+    const CTransaction* GetConflictTx(const COutPoint& prevout) const EXCLUSIVE_LOCKS_REQUIRED(cs);
+
+    /** Translate a set of hashes into a set of pool iterators to avoid repeated lookups */
+    setEntries GetSetIter(const std::set<uint256>& hashes) const EXCLUSIVE_LOCKS_REQUIRED(cs);
```

Comment:
Nit: the name sounds like it's a getter + setter operator. Maybe call it `GetIterSet`?

Reply:
Renamed to `GetIterSet` according to your feedback